core/tests/puppeteer-acceptance-tests/blogpostAdmin-tests/blog-post-create-draft.js

```diff
@@ -0,0 +1,19 @@
+const e2eBlogPostAdmin = require("../utility-functions/blogPostAdminUtils.js");
```

Comment:
Do not rename "blog post admin utils" to "blog post admin" -- a collection of tools (implied by "utils") is not the same conceptually as a user.

I suggest doing something like: `import { BlogPostAdmin } from '/users/blogPostAdmin.model` -- similar to what we do in the rest of our codebase. Is that possible?

Also, just to check, why are we using JS rather than TS? Is TS possible?